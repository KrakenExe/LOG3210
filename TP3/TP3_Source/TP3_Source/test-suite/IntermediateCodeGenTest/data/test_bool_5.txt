Declaration
bool a;
bool b;
bool c;
EndDeclaration

a = b && c;
