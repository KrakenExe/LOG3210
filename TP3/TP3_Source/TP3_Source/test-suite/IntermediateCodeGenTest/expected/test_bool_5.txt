if b == 1 goto _L3
goto _L2
_L3
if c == 1 goto _L1
goto _L2
_L1
a = 1
goto _L0
_L2
a = 0
_L0
